CRM/Event/Form/Participant.php

```diff
@@ -1487,14 +1456,7 @@ protected function preparePaidEventProcessing($params): array {
       }
     }
 
-    if (isset($params['priceSetId'])) {
-      if (!empty($lineItem[0])) {
-        $this->set('lineItem', $lineItem);
-        $this->_lineItem = $lineItem;
-      }
```

Comment:
@eileenmcnaughton I'm confused about how the keys match up here. This code makes it look like `$this->_lineItem` is a non-associative array (because L1491 asserts that it has a `[0]` index)